CITATION.cff

```diff
@@ -0,0 +1,41 @@
+cff-version: 1.2.0
+message: "Please cite this software using these metadata."
+type: software
+title: Shapely
+version: "2.0"
+# date-released: "2022-MM-DD"
+# doi: 10.5281/zenodo.#######
+abstract: "Manipulation and analysis of geometric objects in the Cartesian plane."
+repository-artifact: https://pypi.org/project/Shapely
+repository-code: https://github.com/shapely/shapely
+license: "BSD-3-Clause"
+authors:
```

Comment:
There is a way to add `- name: "Other Shapely contributors"` at the end, but I'm not sure if it adds value. I'll add it for now.

Reply:
I'm for whatever helps people cite Shapely. If publishing about Shapely in a journal would help the project, I'd be down for that, too. It's a significant project, for sure.

Reply:
This is just a software citation, so nothing too fancy or any [impact](https://en.wikipedia.org/wiki/Citation_impact). I don't think Google Scholar picks up on these.

An actual journal citation could take precedence, but would take lots of effort to prepare. A few recent high-impact examples are [NumPy in _Nature_](https://doi.org/10.1038/s41586-020-2649-2) and [SciPy in _Nature Methods_](https://doi.org/10.1038/s41592-019-0686-2), each with several thousand citations per year. I'm down with this too, but probably best to discuss separately, perhaps after 2.0 has landed.

---

Looking at the outputs with  "Other Shapely contributors":
> `@misc`{YourReferenceHere,
author = {Gillies, Sean and van der Wel, Casper and Van den Bossche, Joris and Taves, Mike W. and Arnott, Joshua and Tonnhofer, Oliver and Ward, Brendan C. and Other Shapely contributors},
title = {Shapely},
url = {https://github.com/shapely/shapely}
}
> Gillies S., van der Wel C., Van den Bossche J., Taves M.W., Arnott J., Tonnhofer O., Ward B.C., Other Shapely contributors Shapely (version 2.0). URL: https://github.com/shapely/shapely

Note the date published was commented out, which is why "(2022)" doesn't appear between authors and title. But I generally find "Other Shapely contributors" a bit out of place, so I might remove. Either way, this would usually be "Gillies et al." in text.

Reply:
Regardless of how it looks like, I think we should keep the "and others" or something like that, because shapely is truly a product of many small contributions. Leaving out the "others" just ignores that important fact.

I would shorten it to `others` and not `Other Shapely Contributers`.

> This is just a software citation, so nothing too fancy or any [impact](https://en.wikipedia.org/wiki/Citation_impact). I don't think Google Scholar picks up on these.

If we would enable Zenodo, Google Scholar does pick these citations up, see for example https://scholar.google.com/citations?view_op=view_citation&hl=nl&user=wceWaeIAAAAJ&citation_for_view=wceWaeIAAAAJ:kNdYIx-mwKoC .

Reply:
I have no problem adding "others" to the last author. I've also noticed the SciPy citation that I linked above has the last author "SciPy 1.0 Contributors", so a catch-all meta-author is not out of place. And good to confirm that these citations actually get picked up in Google Scholar.